dashboard/app/controllers/registrations_controller.rb

```diff
@@ -23,18 +23,22 @@ def update
         @user.update_without_password(update_params(params))
       end
 
+    account_update_response(successfully_updated, @user)
+  end
+
+  def account_update_response(successfully_updated, user)
```

Comment:
Should this be private? 

nit: have a verb in the method name, maybe something like `respond_to_account_update` ?